src/app/api/models/portfolio-submission.coffee

```diff
@@ -1,4 +1,7 @@
-angular.module("doubtfire.api.models.portfolio-submission", [
+_ = require('lodash')
+ngFileUpload = require('ng-file-upload')
```

Comment:
Correct, was mixing this up with angular-file-upload which was deprecated by our common/file-uploader directive a while back

On 7 Aug 2016, at 4:48 PM, John <notifications@github.com> wrote:
> 
> In src/app/api/models/portfolio-submission.coffee <https://github.com/doubtfire-lms/doubtfire-web/pull/34#discussion_r73796751>:
> 
> > @@ -1,4 +1,7 @@
> > -angular.module("doubtfire.api.models.portfolio-submission", [
> > +_ = require('lodash')
> > +ngFileUpload = require('ng-file-upload')
> There are references to ngf-* directives in common/file-uploader which in turn seems to be referenced elsewhere throughout the app. possibly angular-file-upload is the deprecated one but it was ambiguous enough that I decided not to touch it
> 
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub <https://github.com/doubtfire-lms/doubtfire-web/pull/34/files/2465065b864b772d6f34e8733f215c21603d996b#r73796751>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ABXUQBXSQLMiO6n7rVBT_eXVfRdyME5Tks5qdX-mgaJpZM4JeWks>.
>